corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -197,6 +197,31 @@ FreeTextEntry.prototype.onPreProcess = function (newValue) {
     this.question.error(this.getErrorMessage(newValue));
 };
 
+function AddressEntry(question, options) {
+    var self = this;
+    FreeTextEntry.call(this, question, options);
+    this.templateType = 'address';
+    this.lengthLimit = options.lengthLimit;
+
+    self.afterRender = function () {
+        var accessToken = 'pk.eyJ1Ijoia25ndXllbi1kaW1hZ2kiLCJhIjoiY2tjM3ZtbXltMDF3OTJ5bnd0cjBpeHc4NSJ9.fVcAJi5sxr-vi_UQf2RnkA';
```

Comment:
@knguyen142 is this a private key? If so we need to cycle it immediately.

Reply:
It's not our key it was just copied from their publicly available one for testing purposes https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder/. I knew about it when I committed it.

Reply:
glad to hear, thanks for confirming.

Would be useful to tag that in a comment in the future. Everyone on the team should be on the lookout for potential secret leaks, so it'll prevent alarm from anyone else who runs across it.